### PR TITLE
Apply theme 'books'

### PR DESCRIPTION
**Pull request by ksuess**
See Sphinx "books" theme: https://sphinx-book-theme.readthedocs.io/en/latest/#

This needs

manual fixes for "books" theme
- [ ] collapsed exercise solutions
      :::{admonition} Solution
      :class: toggle -> :class: dropdown
	- [ ] :linenos: true -> :linenos:

and nice to have fixes

- [x] obsolete: intersphinx: references of docs.plone.org https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
- [ ] check /about/glossary: offer two sections: Plone and Non-Plone

### CODE REVIEW

File: about/glossary.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  [Elasticsearch](https://www.elastic.co/elastic-stack/) is an open source enterprise search platform.
+
+Solr
+  [Solr](https://solr.apache.org/) is an open source enterprise search platform.
+
+
+ReactJS
+  A JavaScript library for building user interfaces
+
+:::
```
Comment (stevepiercy) [inline about/glossary.md:45]: This file is superseded by PR #512. After it is merged, and after @stevepiercy moves the global glossary to `docs/glossary.md`, compare to see if anything was missed.

File: about/glossary.rst [hunk 2]
```diff
@@ -1,58 +0,0 @@
...
-
-   Solr
-      `Solr <http://lucene.apache.org/solr/>`_ is a popular, blazing-fast, open source enterprise search platform built on Apache Lucene.
-
-   ZCML
-      The `Zope Configuration Mark-up Language <https://docs.plone.org/develop/addons/components/zcml.html>`_.
-
-   Diazo
-      `Diazo theme engine guide <http://docs.diazo.org/en/latest/>`_
-      Diazo allows you to apply a theme contained in a static HTML web page to a dynamic website created using any server-side technology.
```
Comment (stevepiercy) [inline about/glossary.rst:58]: This and all other `.rst` files are removed in #512. You will need to rebase on `master`, once that PR is merged.

File: advanced-python/Makefile [hunk 3]
```diff
@@ -1,20 +0,0 @@
...
-# Put it first so that "make" without argument is like "make help".
-help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-.PHONY: help Makefile
-
-# Catch-all target: route all unknown targets to Sphinx using the new
-# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```
Comment (stevepiercy) [inline advanced-python/Makefile:20]: This and all other children `Makefile`s are removed in #512.

You will need to rebase your branch on `master` after #512 is merged.

File: advanced-python/conf.py [hunk 4]
```diff
@@ -1,173 +0,0 @@
...
-# (source start file, target name, title, author,
-#  dir menu entry, description, category)
-texinfo_documents = [
-    (master_doc, 'webframeworkfromscracth', 'webframework from scracth Documentation',
-     author, 'webframeworkfromscracth', 'One line description of project.',
-     'Miscellaneous'),
-]
-
-
-
```
Comment (stevepiercy) [inline advanced-python/conf.py:173]: This and all other children `conf.py`s are removed in #512.

You will need to rebase your branch on `master` after #512 is merged.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I started a review, but realized that we duplicated each other's efforts in many places, so I paused my review. My PR #515 incorporated changes from #511 and #512, and duplicate many of your changes. I think that you should rebase your branch on `master` after #515 is reviewed and merged to `master`, or you can rebase your branch on `myst-convert-all` now.

I will resume my review after #515 is reviewed, approved, and merged, and this branch is rebased on `master`.

**Comment by ksuess** [comment]
This PR is obsolete and will be replaced with PR based on https://github.com/plone/training/pull/515

**Pull request closed without merging**